test/benchee_test.exs

```diff
@@ -225,6 +226,73 @@ defmodule BencheeTest do
     refute output =~ ~r/compar/i
   end
 
+  test "integration profiling can be deactivated" do
+    output =
+      capture_io(fn ->
+        Benchee.run(
+          %{"Sleeps" => fn -> :timer.sleep(10) end},
+          Keyword.merge(
+            @test_configuration,
+            profile_after: false
+          )
+        )
+      end)
+
+    refute output =~ ~r/Profiling.+with/
+    refute output =~ ~r/Profile done/
```

Comment:
small thing: I like to make these regexes ignore case (just add `i` after the last `/`) to make them a little more robust at "no cost"